common/reftest-wait.js

```diff
@@ -1,7 +1,18 @@
+/**
+ * Remove the `reftest-wait` class on the document element.
+ * The reftest runner will wait with taking a screenshot while
+ * this class is present.
+ *
+ * See https://web-platform-tests.org/writing-tests/reftests.html#controlling-when-comparison-occurs
+ */
 function takeScreenshot() {
     document.documentElement.classList.remove("reftest-wait");
 }
 
+/**
+ * Call `takeScreenshot()` after a timeout.
```

Comment:
Might be worth noting that - as with setTimeout - we will delay for *at least* |timeout| milliseconds, but may delay for more

Reply:
```suggestion
 * Call `takeScreenshot()` after a delay of at least |timeout| milliseconds.
```